_events/2016-09-24.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Noise Hackathon"
+description: "The Noise Hackathon centers on the notion of “Noise in 3D: Data-Driven, Art-Driven & Community-Driven” while embracing both the vastness and idiosyncrasies of the world of noise. Talks, panels, and workshops will provide gateways into the complexities of noise including urban noise pollution and noise music to jump-start a full day of noise hacking. Please join us in making some noise."
```

Comment:
I don't think these old events matter much because we're just backfilling them for completeness but... 

The 'description' field gets currently gets shown on the 'Past Events' page so maybe put this full description in the body and use something like

*"Talks, panels, and workshops will provide gateways into the complexities of noise including urban noise pollution and noise music"*

for the description.

<img width="783" alt="screen shot 2017-04-17 at 22 12 57" src="https://cloud.githubusercontent.com/assets/891252/25111812/3a456170-23bb-11e7-953c-57e2bf5c82b9.png">